docs/mesos/RpmCreateInstallCalico.md

```diff
@@ -0,0 +1,28 @@
+# Create and Install the Calico Mesos Plugin RPM
+This guide will walk you through building and installing the Calico-Mesos plugin from an RPM that can be built from the [calico-mesos repository](https://github.com/projectcalico/calico-mesos). The RPM will automate the installation of everything needed to run Calico in your Mesos cluster. 
```

Comment:
`...everything needed to use Calico for networking in your Mesos Cluster.` The way it currently reads sounds like Calico is a Mesos app.